test/controllers/update/beneficial-owner-bo-mo-review.controller.spec.ts

```diff
@@ -0,0 +1,91 @@
+jest.mock("ioredis");
+jest.mock("../../../src/utils/logger");
+jest.mock('../../../src/middleware/authentication.middleware');
+jest.mock('../../../src/middleware/company.authentication.middleware');
+jest.mock('../../../src/middleware/service.availability.middleware');
+jest.mock('../../../src/middleware/navigation/update/has.presenter.middleware');
+jest.mock('../../../src/utils/application.data');
+
+import { NextFunction, Request, Response } from "express";
+import { beforeEach, expect, jest, test, describe } from "@jest/globals";
+import request from "supertest";
+import { serviceAvailabilityMiddleware } from "../../../src/middleware/service.availability.middleware";
+
+import * as config from "../../../src/config";
+import app from "../../../src/app";
+import {
+  ANY_MESSAGE_ERROR,
+  PAGE_TITLE_ERROR,
+  SERVICE_UNAVAILABLE,
+} from "../../__mocks__/text.mock";
+
+import { APPLICATION_DATA_MOCK } from "../../__mocks__/session.mock";
+
+import { getApplicationData } from "../../../src/utils/application.data";
+import { authentication } from "../../../src/middleware/authentication.middleware";
+import { companyAuthentication } from "../../../src/middleware/company.authentication.middleware";
+import { hasUpdatePresenter } from "../../../src/middleware/navigation/update/has.presenter.middleware";
+import { logger } from "../../../src/utils/logger";
+
+const pageTitle = "You're about to review beneficial owner and managing officer information";
+
+const mockHasUpdatePresenter = hasUpdatePresenter as jest.Mock;
+mockHasUpdatePresenter.mockImplementation((req: Request, res: Response, next: NextFunction) => next() );
+
+const mockAuthenticationMiddleware = authentication as jest.Mock;
+mockAuthenticationMiddleware.mockImplementation((req: Request, res: Response, next: NextFunction) => next() );
+
+const mockCompanyAuthenticationMiddleware = companyAuthentication as jest.Mock;
+mockCompanyAuthenticationMiddleware.mockImplementation((req: Request, res: Response, next: NextFunction) => next() );
+
+const mockServiceAvailabilityMiddleware = serviceAvailabilityMiddleware as jest.Mock;
+mockServiceAvailabilityMiddleware.mockImplementation((req: Request, res: Response, next: NextFunction) => next() );
+
+const mockGetApplicationData = getApplicationData as jest.Mock;
+const mockLoggerDebugRequest = logger.debugRequest as jest.Mock;
+
+describe("BENEFICIAL OWNER BO MO REVIEW controller", () => {
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  describe("GET tests", () => {
+    test(`renders the ${config.BENEFICIAL_OWNER_BO_MO_REVIEW_PAGE} page`, async () => {
+      mockGetApplicationData.mockReturnValue({ ...APPLICATION_DATA_MOCK });
+
+      const resp = await request(app).get(config.BENEFICIAL_OWNER_BO_MO_REVIEW_URL);
+      expect(resp.status).toEqual(200);
+      expect(resp.text).toContain(pageTitle);
+      expect(resp.text).not.toContain(PAGE_TITLE_ERROR);
+      expect(resp.text).toContain("Overseas Entity Name");
```

Comment:
Let's use the mock variables that exist for these values e.g. `OVERSEAS_NAME_MOCK`

Reply:
Done.